x-pack/plugins/siem/public/containers/kpi_network/index.gql_query.ts

```diff
@@ -7,6 +7,14 @@
 import gql from 'graphql-tag';
 
 export const kpiNetworkQuery = gql`
+  fragment KpiNetworkChartFields on KpiNetworkHistogramData {
```

Comment:
nit - it will be nice to have some logic in our server side, so we only bring one count.